.editorconfig

```diff
@@ -8,13 +8,8 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
-indent_size = 4
-trim_trailing_whitespace = true
-
-# Matches the exact files either package.json or .travis.yml
-[{package.json, .travis.yml}]
-indent_style = space
 indent_size = 2
```

Comment:
Isn't this also a default config?

Reply:
In `EditorConfig`?

I couldn't find anything at https://editorconfig.org that shows that it has a default.

Reply:
Ah, no. I confused it with the prettier defaults. All good.